api/src/main/java/org/opencord/aaa/AaaConfig.java

```diff
@@ -85,7 +85,19 @@
     // Packet Customizer Default value
     protected static final String DEFAULT_PACKET_CUSTOMIZER = "default";
 
-
+    //Event publisher schedule value initial delay
+    private static final int INITIAL_DELAY = 0;
```

Comment:
I felt the INITIAL_DELAY is not required and can be avoided. The same value of Repeat_Delay can be used for this initial delay or a initial delay of 0 can be used.